scss/mixin.scss

```diff
@@ -43,3 +43,20 @@
     }
   }
 }
+
+@mixin form-control( $border-color ) {
+  @include scaleText(16px, 24px, 20px, 28px);
+
+  border-radius: 0;
+  padding: 0.75rem;
+  border-color: $border-color;
+
+  &:focus,
+  &:active {
+    border-color: $light-blue;
+  }
+
+  @media (min-width: $bp-md) {
```

Comment:
Following the above thought, a media query on a generic mixin feels ripe for bugs

Reply:
See https://github.com/mozilla/network-pulse/pull/1101#discussion_r302239629